demo/ARContainerViewController.swift

```diff
@@ -51,16 +51,20 @@ final class ARContainerViewController: UIViewController {
         navigationService?.start()
     }
 
+    func configureFor(arManager: VisionARManager?) {
+        arViewController.set(arManager: arManager)
+    }
+
     func present(sampleBuffer: CMSampleBuffer) {
-        arViewController.present(sampleBuffer: sampleBuffer)
+//        arViewController.present(sampleBuffer: sampleBuffer)
```

Comment:
Should we remove these methods altogether?

Reply:
my fault, will remove

Reply:
fixed